mam4/CMakeLists.txt

```diff
@@ -2,8 +2,8 @@
 # since it doesn't have a reasonable build system.
 
 # download and extract the source for MAM4
-set(MAM4_SOURCE_URL https://github.com/AMBRS-project/MAM_box_model/archive/refs/heads/main.zip)
-set(MAM4_SOURCE_DIR ${CMAKE_CURRENT_BINARY_DIR}/MAM_box_model-main)
+set(MAM4_SOURCE_URL https://github.com/Duncan-Quevedo/MAM_box_model_dq/archive/refs/heads/main.zip)
```

Comment:
Maybe you could push your MAM4 branch to the main repo and we could refer to it here?